Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM node:14-slim as builder
+FROM node:18-slim as builder
+
+ARG NODE_ENV=production
+ENV NODE_ENV=$NODE_ENV
```

Comment:
These should ensure that Webpack etc. isn't installed into the deployed image. It will install React, which isn't needed by the proxy server, so can probably be improved.